fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/mixin/registry/sync/MixinIdList.java

```diff
@@ -86,15 +86,15 @@ public void fabric_remapId(int from, int to) {
 	@Override
 	public void fabric_remapIds(Int2IntMap map) {
 		// remap idMap
-		idMap.replaceAll((a, b) -> map.get(b));
+		idMap.replaceAll((a, b) -> map.get((int) b));
```

Comment:
IDE was complaining about get(Object) being deprecated in the FastUtils collection,  hence the explicit cast here.